src/lib/dialog/dialog-config.ts

```diff
@@ -57,6 +57,18 @@ export class MatDialogConfig {
   /** Height of the dialog. */
   height?: string = '';
 
+  /** Min-width of the dialog. */
+  minWidth?: string;
+
+  /** Min-height of the dialog */
+  minHeight?: string;
+
+  /** Max-width of the dialog. */
+  maxWidth?: string = '80vw';
+
+  /** Max-height of the dialog */
+  maxHeight?: string;
```

Comment:
These should all be `string | number`, where a number implies a pixel value to match the OverlayConfig values

Reply:
Done.